app/code/core/Mage/Catalog/Model/Product/Attribute/Backend/Groupprice/Abstract.php

```diff
@@ -55,14 +55,23 @@ abstract protected function _getDuplicateErrorMessage();
     /**
      * Retrieve websites currency rates and base currency codes
      *
+     * @param int $websiteId
```

Comment:
I think PHPStan will complain about this line not including null as a possible value for `$websiteId`.

Reply:
absolutely, just fixed it!

Reply:
It didn't! PHPStan feeling nice today :D

Reply:
:-D maybe it will complain now that I changed it ;-)